rapid/workflow/queue_handlers/handlers/standard_queue_handler.py

```diff
@@ -0,0 +1,109 @@
+                    try:
+                        response = client.send_work(work_request, self.rapid_config.verify_certs)
+                        if response.status_code == 423:
+                            client.sleep = True
+                            raise Exception("Client was busy, can't take work.")
+                        if response.status_code == 201:
+                            if 'X-Exclude-Resource'.lower() in response.headers:
+                                client.sleep = True
+                        else:
+                            logger.info("Client didn't respond right: {} returned {}".format(client.ip_address, response.status_code))
+
+                        break  # Break, we sent work to the other client
+                    except Exception as exception:
+                        logger.error("Could not send work to worker: [{}]".format(str(exception)))
+                        self.action_instance_service.edit_action_instance(work_request.action_instance_id, {"status_id": StatusConstants.READY,
+                                                                                                            "start_date": None,
+                                                                                                            "assigned_to": None})
+                except ConnectTimeout:
+                    # Should reset, this is a problem, server not there.
+                    self.action_instance_service.edit_action_instance(work_request.action_instance_id, {"status_id": StatusConstants.READY,
+                                                                                                        "start_date": None,
+                                                                                                        "assigned_to": None})
+                except ReadTimeout as read_timeout:
+                    logger.error(read_timeout)
+                    logger.error(client.get_work_uri())
+                except ConnectionError as error:
+                    logger.error(error)
+                    logger.error(client.get_work_uri())
+                except Exception as exception:
+                    logger.error(exception)
+                    logger.error(client.get_work_uri())
+        pages = None
+        clients_array = None
+
+    def __init__(self, rapid_config, action_instance_service):
+        """
+        :param rapid_config:
+        :type rapid_config: rapid.master.master_configuration.MasterConfiguration
+        :param action_instance_service:
+        :type action_instance_service: rapid.workflow.action_instance_service.ActionInstanceService
+        """
+        super(StandardQueueHandler, self).__init__(rapid_config)
+        self.action_instance_service = action_instance_service
+
+    def can_process_work_request(self, work_request):
+        return len(self._get_grain_type_split(work_request.grain)) < 2
```

Comment:
anything that can't be split by "://", this handler takes. Which is all our current grains.